projects/observability/src/pages/apis/service-detail/instrumentation/components/instrumentation-details.component.ts

```diff
@@ -44,4 +66,32 @@ export class InstrumentationDetailsComponent {
   public onClickBack(): void {
     this.router.navigate(['../'], { relativeTo: this.route });
   }
+
+  public getHeaderIcon(score: number): string {
+    if (score < 50) {
+      return IconType.Close;
+    }
+
+    if (score < 70) {
+      return IconType.Warning;
+    }
+
+    return IconType.Checkmark;
+  }
+
+  public getIconColor(score: number): string {
+    return this.serviceInstrumentationService.getColorForScore(score).dark;
+  }
```

Comment:
If we are leveraging the same method `getColorForScore`, do you think we can update that method to return the `color.dark` thing itself?

Reply:
Not sure. That method was originally created for ProgressCircle, which requires both light and dark shades of the color. So the return value is an object that contains both shades of, say, amber: `{ light: '#ffecbc', dark: '#ffa01c' }`

<img width="182" alt="Screenshot 2022-09-30 at 3 24 39 PM" src="https://user-images.githubusercontent.com/29686866/193245149-43da66a7-1781-4a90-8b8f-827a0707baaf.png">

ProgressCircle is the only component that needs both shades. In all other places, we just need the dark shade. How can we refactor it?

Reply:
Ahh, any approach won't be clean then, resolving.